src/transformers/__init__.py

```diff
@@ -39,6 +39,20 @@
 
 from typing import TYPE_CHECKING
 
+
+# Integrations: import before any ML framework
+# isort: off
+from .integrations import (
+    is_comet_available,
```

Comment:
No this can't be imported here: the init is written so that no framework is imported inside to speed up the import and to only import the frameworks for a user on a need-to basis (e.g. if someone does `from transformers import BertModel`, PyTorch is imported, but not TensorFlow, Flax, comet_ml, wandb etc.)

Reply:
Ok, thanks for the feedback. We'll come up with a better way to solve the problem.